src/bos/reporter/status_reporter/__main__.py

```diff
@@ -65,7 +65,7 @@ def report_state_until_success(component):
         session = requests_retry_session()
         try:
             bss_referral_token = get_value_from_proc_cmdline('BSS_REFERRAL_TOKEN')
-            state = {'boot_artifacts_record': {'bss_referral_token': bss_referral_token}}
```

Comment:
I'm not sure if you actually want this change in this PR/commit since it doesn't have to do with the v2 api naming.

Reply:
Removed.